src/download-file.js

```diff
@@ -19,6 +21,11 @@ const downloadFile = (file) => {
   return new Promise((resolve, reject) => {
     fs.stat(fileName, (err) => {
       if (err) { // File doesn't exists yet
+        const { dir } = parsePath(fileName);
+
+        // TODO: Change synchronous call.
+        fse.ensureDirSync(dir);
```

Comment:
Is it possible to use an async function here instead of an arrow function? Maybe that way you could:
```javascript
await fse.ensureDir(dir);
```